release-notes/3.0/preview/api-diff/preview8/ASP.Net/3.0-preview8.md

```diff
@@ -0,0 +1,41 @@
+* [Microsoft.AspNetCore.Components.Browser](3.0-preview8_Microsoft.AspNetCore.Components.Browser.md)
+* [Microsoft.AspNetCore.Components.Browser.Rendering](3.0-preview8_Microsoft.AspNetCore.Components.Browser.Rendering.md)
+* [Microsoft.AspNetCore.Components.CompilerServices](3.0-preview8_Microsoft.AspNetCore.Components.CompilerServices.md)
+* [Microsoft.AspNetCore.Components.Forms](3.0-preview8_Microsoft.AspNetCore.Components.Forms.md)
+* [Microsoft.AspNetCore.Components.Layouts](3.0-preview8_Microsoft.AspNetCore.Components.Layouts.md)
+* [Microsoft.AspNetCore.Components.Rendering](3.0-preview8_Microsoft.AspNetCore.Components.Rendering.md)
+* [Microsoft.AspNetCore.Components.RenderTree](3.0-preview8_Microsoft.AspNetCore.Components.RenderTree.md)
+* [Microsoft.AspNetCore.Components.Routing](3.0-preview8_Microsoft.AspNetCore.Components.Routing.md)
+* [Microsoft.AspNetCore.Components.Server](3.0-preview8_Microsoft.AspNetCore.Components.Server.md)
+* [Microsoft.AspNetCore.Components.Web](3.0-preview8_Microsoft.AspNetCore.Components.Web.md)
+* [Microsoft.AspNetCore.Connections](3.0-preview8_Microsoft.AspNetCore.Connections.md)
+* [Microsoft.AspNetCore.DataProtection.Cng.Internal](3.0-preview8_Microsoft.AspNetCore.DataProtection.Cng.Internal.md)
+* [Microsoft.AspNetCore.DataProtection.Internal](3.0-preview8_Microsoft.AspNetCore.DataProtection.Internal.md)
+* [Microsoft.AspNetCore.Hosting.Server.Abstractions](3.0-preview8_Microsoft.AspNetCore.Hosting.Server.Abstractions.md)
+* [Microsoft.AspNetCore.Http](3.0-preview8_Microsoft.AspNetCore.Http.md)
+* [Microsoft.AspNetCore.Http.Features](3.0-preview8_Microsoft.AspNetCore.Http.Features.md)
+* [Microsoft.AspNetCore.Identity](3.0-preview8_Microsoft.AspNetCore.Identity.md)
+* [Microsoft.AspNetCore.Mvc.ActionConstraints](3.0-preview8_Microsoft.AspNetCore.Mvc.ActionConstraints.md)
+* [Microsoft.AspNetCore.Mvc.Diagnostics](3.0-preview8_Microsoft.AspNetCore.Mvc.Diagnostics.md)
+* [Microsoft.AspNetCore.Mvc.Infrastructure](3.0-preview8_Microsoft.AspNetCore.Mvc.Infrastructure.md)
+* [Microsoft.AspNetCore.Mvc.ViewFeatures](3.0-preview8_Microsoft.AspNetCore.Mvc.ViewFeatures.md)
+* [Microsoft.AspNetCore.Routing.Internal](3.0-preview8_Microsoft.AspNetCore.Routing.Internal.md)
+* [Microsoft.AspNetCore.Routing.Template](3.0-preview8_Microsoft.AspNetCore.Routing.Template.md)
+* [Microsoft.AspNetCore.Routing.Tree](3.0-preview8_Microsoft.AspNetCore.Routing.Tree.md)
+* [Microsoft.AspNetCore.SignalR](3.0-preview8_Microsoft.AspNetCore.SignalR.md)
+* [Microsoft.Extensions.Hosting.Internal](3.0-preview8_Microsoft.Extensions.Hosting.Internal.md)
+* [Microsoft.JSInterop](3.0-preview8_Microsoft.JSInterop.md)
+* [Microsoft.JSInterop.Internal](3.0-preview8_Microsoft.JSInterop.Internal.md)
+* [System.Drawing](3.0-preview8_System.Drawing.md)
+* [System.Drawing.Printing](3.0-preview8_System.Drawing.Printing.md)
+* [System.IO.Pipelines](3.0-preview8_System.IO.Pipelines.md)
```

Comment:
The api diff for pipelines shows up twice now (once for standalone, and once here).

Reply:
I think Asp.net is shipping this package as a part of their framework while .net core ship this as a standalone package

Reply:
@Anipik exactly right.

Like Microsoft.WindowsDesktop.App, Microsoft.AspNetCore.App assemblies have a handful of CoreFx dependencies that aren't in Microsoft.NETCore.App. We ship with them in our shared framework contributions. (Pity Microsoft.AspNetCore.App and Microsoft.WindowsDesktop.App contain a couple of duplicate assemblies but those two sides of the shared framework definitely should not depend on each other.)